docs/release-notes.asciidoc

```diff
@@ -7,6 +7,29 @@
 :issue: https://github.com/elastic/kibana/issues/
 :pull: https://github.com/elastic/kibana/pull/
 
+[discrete]
+[[release-notes-7.13.2]]
+== 7.13.2
+
+[discrete]
+[[known-issue-7.13.2]]
+==== Known issue
+The following ML rules contain incorrectly configured ML job IDs (underscores were used instead of dashes between words) and cannot be successfully activated after they are enabled. Running these rules will cause an error message to display, indicating that an error occurred during the rule's execution. This issue is present in 7.13, 7.13.1, and 7.13.2. ({issue}102146[#102146]) 
```

Comment:
```suggestion
The following {ml-app} rules contain incorrectly configured ML job IDs (underscores were used instead of dashes between words) and cannot be successfully activated after they are enabled. Running these rules will cause an error message to display, indicating that an error occurred during the rule's execution. This issue is present in {stack} 7.13, 7.13.1, and 7.13.2. ({issue}102146[#102146]) 
```